docs/concepts/aws.md

```diff
@@ -28,3 +28,17 @@ spec:
     - ipBlock:
         cidr: 0.0.0.0/0
 ```
+
+## TLS handshake errors in kube-apiserver logs
+
+On AWS platform, you may see the following logs coming from `kube-apiserver` pods:
+
+```
+I0408 05:35:02.865305       1 log.go:172] http: TLS handshake error from 127.0.0.1:45332: read tcp 127.53.210.227:7443->127.0.0.1:45332: read: connection reset by peer
+I0408 05:35:12.865457       1 log.go:172] http: TLS handshake error from 127.0.0.1:45424: read tcp 127.53.210.227:7443->127.0.0.1:45424: read: connection reset by peer
+I0408 05:35:22.865279       1 log.go:172] http: TLS handshake error from 127.0.0.1:45516: read tcp 127.53.210.227:7443->127.0.0.1:45516: read: connection reset by peer
+```
+
+Those logs are harmless and are caused by AWS ELB probing `kube-apiserver` for availability. Unfortunately, AWS ELB does not support authentication for probe requests, so we must use TCP probes, which causes exactly those errors.
```

Comment:
```suggestion
These errors are harmless and are caused by an AWS ELB probing `kube-apiserver` for availability. The probe succeeds but AWS ELBs do not support authentication for probe requests so `kube-apiserver` complains with error logs.
```

Reply:
This does not seem logical to me to mention authentication if we change it like this.

Reply:
Rephrased. Is it better now?